src/module.c

```diff
@@ -10141,6 +10159,9 @@ int moduleUnload(sds name) {
     } else if (module->blocked_clients) {
         errno = EAGAIN;
         return C_ERR;
+    } else if (moduleHoldsTimer(module)) {
+        errno = EINPROGRESS;
```

Comment:
There is a comment describing the different errno values before this function. This one can be added to the list.

In moduleCommand, where moduleUnload is called, there is a `switch(errno)` where this new errno can be added to have a specific ERR message for this case.

Reply:
thanks for your advice, will do

Reply:
OK, looks good. It can also be added in this comment: https://github.com/redis/redis/blob/9b7576737f61cc37b3007636e18914c2d6f60a42/src/module.c#L10142

Reply:
It seems that you forgot to add EINPROGRESS to the top comment as @zuiderkwast  said.

Reply:
Thanks for the reminder, seems missed that since it folded